### PR TITLE
[main] BXMSPROD-1539 composite actions centralized github actions

### PR DESCRIPTION
**Pull request by Ginxo**

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -16,35 +16,25 @@ jobs:
...
-        run: |
-          df -h
-          sudo rm -rf /usr/local/lib/android
-          docker rmi $(docker image ls -aq)
-          df -h
-      - name: Setup Maven And Java Version
-        uses: s4u/setup-maven-action@v1.2.1
+        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/long-paths@main
+      - name: Java and Maven Setup
+        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/java@main
```
Comment (radtriste) [inline .github/workflows/pull_request.yml:25]: @Ginxo you are going to need to update this

Comment (Ginxo) [reply .github/workflows/pull_request.yml:25]: by what sorry, I don't get it... by kogito-pipelines one? @radtriste

Comment (radtriste) [reply .github/workflows/pull_request.yml:25]: we will need to use the `kiegroup/droolsjbpm-build-bootstrap/.ci/actions/maven@main` action

Comment (Ginxo) [reply .github/workflows/pull_request.yml:25]: we will align kogito-pipelines composite actions and droolsjbpm-build-bootstrap soon, yes. My question is more about "do we still pointing to droolsjbpm-build-bootstrap or we point to kogito-pipelines?" @radtriste

Comment (radtriste) [reply .github/workflows/pull_request.yml:25]: oh sorry misread the url...
optawebs are now on Kogito pipelines for the main branch, so we should get the pr config from kogito-pipelines ;)

**Pull request merged**